src/hook/useTurnOnAllNotificationsOnAppStartForAndroid.ts

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {Platform} from 'react-native';
+import {usePushTopics} from './usePushTopics';
+import {useIsFirstAppStart} from 'src/hook/useIsFirstAppStart';
+
+export function useTurnOnAllNotificationsOnAppStartForAndroid() {
+  const {toggleTopic, topics, isLoading: topicsLoading} = usePushTopics();
+  const {data: isFirstAppStart, isLoading: firstAppStartLoading} = useIsFirstAppStart();
+
+  const isRunnedOnceRef = React.useRef(false);
```

Comment:
Why is this ref needed? isn't `isFirstAppStart` enough?

Reply:
`isFirstAppStart` stays true the whole time user has opened the app for the first time. I did it like that, so it could be used in different hooks if needed. therefor, if I don't do this check toggling the topics would cause the effect to run again and do the same thing over and over.